node/test/Test/Pos/CborSpec.hs

```diff
@@ -347,197 +304,51 @@ testAgainstFile name x expected =
-                prop "Signature a" (soundInstanceProperty @(Signature U))
-                prop "Signed a"    (soundInstanceProperty @(Signed U))
-                prop "RedeemSignature a"    (soundInstanceProperty @(RedeemSignature U))
-                prop "ProxySecretKey w"     (soundInstanceProperty @(ProxySecretKey U))
-                prop "ProxySignature w a"   (soundInstanceProperty @(ProxySignature U U))
-                prop "AbstractHash SHA256"  (soundInstanceProperty @(AbstractHash SHA256 U))
-                prop "DataMsgSKLight" (soundInstanceProperty @(DataMsg ProxySKLight))
-                prop "DataMsgSKHeavy" (soundInstanceProperty @(DataMsg ProxySKHeavy))
-                prop "DataMsgSKLightConfirmation" (soundInstanceProperty @(DataMsg ProxySKLightConfirmation))
-                prop "BackupPhrase" (soundInstanceProperty @BackupPhrase)
-                prop "PrevValue a"  (soundInstanceProperty @(PrevValue U))
-                -- Pending specs which doesn't have an `Arbitrary` or `Eq` instance defined.
-                it "UserSecret" $ pendingWith "No Eq instance defined"
-                it "WalletUserSecret" $ pendingWith "No Eq instance defined"
-                pendingNoArbitrary "Undo"
-                pendingNoArbitrary "DataMsg (UpdateProposal, [UpdateVote])"
-                pendingNoArbitrary "DataMsg UpdateVote"
-                pendingNoArbitrary "MsgGetHeaders"
-                pendingNoArbitrary "MsgGetBlocks"
-                pendingNoArbitrary "WithHash"
-                pendingNoArbitrary "Pvss.PublicKey"
-                pendingNoArbitrary "Pvss.KeyPair"
-                pendingNoArbitrary "Pvss.Secret"
-                pendingNoArbitrary "Pvss.DecryptedShare"
-                pendingNoArbitrary "Pvss.EncryptedShare"
-                pendingNoArbitrary "Pvss.Proof"
-                pendingNoArbitrary "AsBinary VssKeyPair"
-                pendingNoArbitrary "Ed25519.PointCompressed"
-                pendingNoArbitrary "Ed25519.Scalar"
-                pendingNoArbitrary "Ed25519.Signature"
-                pendingNoArbitrary "CC.ChainCode"
-                pendingNoArbitrary "CC.XPub"
-                pendingNoArbitrary "CC.XPrv"
-                pendingNoArbitrary "CC.XSignature"
-                pendingNoArbitrary "EdStandard.PublicKey"
-                pendingNoArbitrary "EdStandard.SecretKey"
-                pendingNoArbitrary "EdStandard.Signature"
-                pendingNoArbitrary "EncryptedSecretKey"
+        describe "Generic deriving" $ do
+            testARecord
+            testAUnit
+            testANewtype
+            binaryTest @ARecord
+            binaryTest @AUnit
+            binaryTest @ANewtype
+        describe "Lib/core instances" $ do
+            binaryTest @(Attributes X1)
+            binaryTest @(Attributes X2)
+            binaryTest @MessageCode
+            binaryTest @SlottingData
```

Comment:
I see some inconsistency in having it here instead of e. g. `Test.Pos.Slotting.BinarySpec`, but it's not critical to me.

Reply:
Oh, yeah, i didn't find a proper place for them. And eventually forgot to think again.